databroker/src/broker.rs

```diff
@@ -1155,6 +1156,7 @@ impl<'a, 'b> DatabaseWriteAccess<'a, 'b> {
             metadata: Metadata {
                 id: temp_id,
                 path: name.clone(),
+                glob_path: name.replace('.', "/"),
```

Comment:
Actually we do care. All clients are configured to use `.` as the path request separator instead of `/`. 
If the `glob_path` string is not added during initialization, it would need to interate over all the signals, replace `.` with `/`, and then checking if each signal matches the request path.

By including the `glob_path` string during initialization, the process is streamlined: it simply translates the request path to the corresponding glob matching path and use it directly. This eliminates the need for repeated replacements and comparisons for each request.

Reply:
I think we need both in case we don't want to make so many changes to the code.
So in case Databroker stores paths **only** with `/` to be able to support the wildcard case, it would mean that all grpc / viss call layer must be alligned to convert the path requested to use `/`.
 
The purpose of adding this `glob_path` string is to use glob_match library **only** for get and subscribe methods (wildcard use cases), meaning we dont have to change and adapt all grpc /viss calls just to solve this specific wildcard use case.